documentation/2.5/themes/hugo-theme-learn/layouts/partials/menu.html

```diff
@@ -66,8 +66,9 @@ <h3>{{ if not $disableShortcutsTitle}}{{ T "Shortcuts-Title"}}{{ end }}</h3>
               {{ end }}
           {{ end }}
         </select>
-        <svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-          width="255px" height="255px" viewBox="0 0 255 255" style="enable-background:new 0 0 255 255;" xml:space="preserve">
+              <svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
```

Comment:
You have a bunch of files that don't need to be part of this PR that are still changing.

Reply:
Revereted the doc changes in [27d2526](https://github.com/oracle/weblogic-kubernetes-operator/pull/2857/commits/27d2526417f5298d3de67b0d2c7eb825d636b01b).